index.html

```diff
@@ -8,20 +8,33 @@
     </head>
     <body class="overflow-hidden">
         <header>
```

Comment:
```suggestion

```
sorry that i didnt spot this before, but this tag is not needed for a navbar and is instead used for headings for content.

Reply:
Made a commit d336443b54664d0e56469a06a8105750746b211b.